library/classes/Prescription.class.php

```diff
@@ -254,7 +257,10 @@ function set_unit($unit) {
     }
 
     function set_id($id) {
-        if (!empty($id) && is_numeric($id)) { $this->id = $id; }
+        //if (!empty($id) && is_numeric($id)) { $this->id = $id; }
+		$phid = explode("-", $id);
+		$id = $phid[0];
+        $this->id = $id;		
```

Comment:
When there was a separate table for the pharmacies, to distinguish the two tables, there was a hyphen in the id. this was to remove the hyphen and get the id. No longer needed since we are going to use the native tables.